content/en/blog/_posts/2021-11-09-non-root-containers-and-devices.md

```diff
@@ -135,14 +135,24 @@ deployments break, an opt-in config entry in both containerd and CRI-O to enable
 defaults to `false` and must be enabled to use the feature.
 
 ## See non-root containers using devices after the fix
-To demonstrate the new behavior, let's use a Data Plane Development Kit (DPDK) application using hardware accelerators, Kubernetes CPU manager, and HugePages as an example. The cluster runs containerd with:
+To demonstrate the new behavior, let's use a Data Plane Development Kit (DPDK) application using hardware accelerators, Kubernetes CPU manager, and HugePages as an example. The cluster runs
+
+Containerd v1 with:
 
 ```toml
 [plugins]
   [plugins."io.containerd.grpc.v1.cri"]
     device_ownership_from_security_context = true
 ```
 
+Containerd v2 with:
```

Comment:
**_If_** it’s decided that this update is necessary, we should also include a note in the article indicating that it has been updated after publication. Please make sure to add such a note somewhere in the article  if we proceed with the changes.